SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2330,12 +2329,12 @@ bool USpatialNetDriver::CreateSpatialNetConnection(const FURL& InUrl, const FUni
 
 bool USpatialNetDriver::HasServerAuthority(Worker_EntityId EntityId) const
 {
-	return StaticComponentView->HasAuthority(EntityId, SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID);
```

Comment:
Static component view used at all anymore? If not, what's left?

Reply:
yeah used in a couple places, the actor channel being one of them. Need to add net owning client worker back to the actor system's data store for that one, which was a little more involved. There's another ticket for dealing with the remaining scv bits: https://improbableio.atlassian.net/browse/UNR-4844